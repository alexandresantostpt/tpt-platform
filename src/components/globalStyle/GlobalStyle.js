import { createGlobalStyle } from 'styled-components'

const GlobalStyle = createGlobalStyle`

    * {
        font-family: Lato, Arial, sans-serif;
    }

    body {
        background: #fafafa;
    }

    #root {
        display: grid;
        grid-column-gap: 0.5rem;
        grid-template:
            'header header header' 70px
            'tasks container summary' calc(100vh - 70px)
            / 25vw calc(50vw - 1.975rem) 25vw;
        max-width: 100vw;
        min-height: 100vh;
    }

    .center {
        display: block;
        text-align: center;
    }
`

export default GlobalStyle
