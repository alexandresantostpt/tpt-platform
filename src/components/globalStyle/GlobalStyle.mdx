---
name: GlobalStyle
route: /global/style
---
import { Playground, PropsTable } from 'docz'

import GlobalStyle from './GlobalStyle'

import '@cfg/scripts'
import '@cfg/style'

# GlobalStyle
<PropsTable of={ GlobalStyle }/>

## Basic usage

```
<GlobalStyle/>
```

<Playground>
    <GlobalStyle/>
</Playground>
