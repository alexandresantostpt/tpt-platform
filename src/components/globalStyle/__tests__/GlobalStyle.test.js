import React from 'react'
import renderer from 'react-test-renderer'

import GlobalStyle from '../GlobalStyle'

describe('Tests for GlobalStyle component', () => {
    it('Should render without crash', () => {
        const tree = renderer.create(<GlobalStyle />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
