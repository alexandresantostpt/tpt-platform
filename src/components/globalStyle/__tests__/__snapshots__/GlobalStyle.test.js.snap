// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for GlobalStyle component Should render without crash 1`] = `null`;
